src/HtmlProcessor/CssVariableEvaluator.php

```diff
@@ -120,8 +122,20 @@ private function replaceVariablesInPropertyValue(string $propertyValue): string
                     )?+
                 \\)
             /x';
-        $callable = \Closure::fromCallable([$this, 'getPropertyValueReplacement']);
-        return (new Preg())->replaceCallback($pattern, $callable, $propertyValue);
+
+        $callable = function (array $matches): string {
+            /** @var array<int, string> $matches */
+            return $this->getPropertyValueReplacement($matches);
+        };
```

Comment:
Having this extra function seems redundant.  Is it not possible to simply have

```php
$callable = [$this, 'getPropertyValueReplacement'];
```

I can't recall offhand why I used `Closure::fromCallable`, but there must have been a reason, and it probably related to Stan.  And I guess that the `Safe` functions not accepting a `Closure` also relates to Stan.

Reply:
Ah, it works if we change the method from `private` to `public`. I'd prefer that as it would allow us to drop the anonymous function.

Would you be okay if I changed that in a follow-up PR (and also made the callback methods `@internal`)?

Reply:
Done and repushed.